components/RelatedItems.vue

```diff
@@ -100,30 +67,42 @@ export default Vue.extend({
     skeletonLength: {
       type: Number,
       default: 5
+    },
+    title: {
+      type: String,
+      default(): string {
+        return this.$t('youMayAlsoLike').toString();
```

Comment:
```suggestion
        return this.$t('youMayAlsoLike');
```

Reply:
This causes an error, because `$t()` technically doesn't return a string.